test/kubemark/start-kubemark-master.sh

```diff
@@ -18,14 +18,32 @@
 sudo docker run --net=host -d gcr.io/google_containers/etcd:2.0.12 /usr/local/bin/etcd --addr=127.0.0.1:4001 --bind-addr=0.0.0.0:4001 --data-dir=/var/etcd/data
 
 # Increase the allowed number of open file descriptors
-+ulimit -n 65536
+ulimit -n 65536
+
+SERVICE_ACCOUNT=("system:scheduler" "system:controller_manager" "system:logging" "system:monitoring" "system:dns")
```

Comment:
I don't think that these token-based service account definitions are still necessary, now that we have per-namespace service accounts. 

@erictune